Fastpack/Reporter.re

```diff
@@ -42,19 +47,14 @@ module Text = {
   };
 
   let report_error = (~ctx: Context.t, ~error: Error.reason) => {
-    let error_msg = Context.string_of_error(ctx, error);
+    let error_msg = Context.stringOfError(ctx, error);
```

Comment:
Do we start mixing snake case with camel case?

Reply:
@TrySound I am slowly converting to camel case